tests/test_assets.py

```diff
@@ -1218,179 +1124,6 @@ def test_error_message_plurality(self,
-        self.assertEqual(cl[0], 0)
-
-        cl = self._get_future_chain('2005-12-21', 'CL')
-        self.assertEqual(cl[0], 1)
-
-        cl = self._get_future_chain('2006-02-01', 'CL')
-        self.assertEqual(cl[-1], 3)
-
-    def test_iter(self):
-        """ Test the __iter__ method of FutureChain.
-        """
-        cl = self._get_future_chain('2005-12-01', 'CL')
-        for i, contract in enumerate(cl):
-            self.assertEqual(contract, i)
-
-        # First contract is now invalid, so sids will be offset by one
-        cl = self._get_future_chain('2005-12-21', 'CL')
-        for i, contract in enumerate(cl):
-            self.assertEqual(contract, i + 1)
-
-    def test_root_symbols(self):
-        """ Test that different variations on root symbols are handled
-        as expected.
-        """
-        # Make sure this successfully gets the chain for CL.
-        cl = self._get_future_chain('2005-12-01', 'CL')
-        self.assertEqual(cl.root_symbol, 'CL')
-
-        # These root symbols don't exist, so RootSymbolNotFound should
-        # be raised immediately.
-        with self.assertRaises(RootSymbolNotFound):
-            self._get_future_chain('2005-12-01', 'CLZ')
-
-        with self.assertRaises(RootSymbolNotFound):
-            self._get_future_chain('2005-12-01', '')
-
-    def test_repr(self):
-        """ Test the __repr__ method of FutureChain.
-        """
-        cl = self._get_future_chain('2005-12-01', 'CL')
-
-        self.assertEqual(
-            repr(cl),
-            "FutureChain('CL', '2005-12-01')"
-        )
-
-    def test_contracts_returns_a_copy(self):
-        cl = self._get_future_chain('2005-12-01', 'CL')
-        self.assertEqual(len(cl), 4)
-
-        contracts = cl.contracts
-        contracts.pop(0)
-
-        self.assertEqual(len(contracts), 3)
-        self.assertEqual(len(cl), 4)
-
-    def test_cme_code_to_month(self):
-        codes = {
-            'F': 1,   # January
-            'G': 2,   # February
-            'H': 3,   # March
-            'J': 4,   # April
-            'K': 5,   # May
-            'M': 6,   # June
-            'N': 7,   # July
-            'Q': 8,   # August
-            'U': 9,   # September
-            'V': 10,  # October
-            'X': 11,  # November
-            'Z': 12   # December
-        }
-        for key in codes:
-            self.assertEqual(codes[key], cme_code_to_month(key))
-
-    def test_month_to_cme_code(self):
-        codes = {
-            1: 'F',   # January
-            2: 'G',   # February
-            3: 'H',   # March
-            4: 'J',   # April
-            5: 'K',   # May
-            6: 'M',   # June
-            7: 'N',   # July
-            8: 'Q',   # August
-            9: 'U',   # September
-            10: 'V',  # October
-            11: 'X',  # November
-            12: 'Z',  # December
-        }
-        for key in codes:
-            self.assertEqual(codes[key], month_to_cme_code(key))
```

Comment:
Open to adding them back in. Had removed them because it was essentially testing whether a `dict` is behaving as expected. (And the unused functions may be an unnecessary step. Following the pattern in other places where we refer to definitions, the definitions could be used as `CME_CODE_TO_MONTH[code]` and `MONTH_TO_CME_CODE[code]`.